guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -191,6 +191,11 @@
                     "type"    : "ENUM",
                     "options" : [ "", "preserve", "unix", "windows" ]
                 },
+                {
+                    "name"    : "clipboard-buffer-size",
+                    "type"    : "ENUM",
+                    "options" : [ "262144", "1048576", "10485760" ]
```

Comment:
There should also be a `""` option to permit the default value to be selected and represented (same for the other protocols).